src/main/scala/cromwell/engine/backend/jes/JesAttributes.scala

```diff
@@ -17,7 +18,8 @@ object JesAttributes {
     "project",
     "baseExecutionBucket",
     "endpointUrl",
-    "maximumPollingInterval"
+    "maximumPollingInterval",
+    "preemptible"
```

Comment:
Hmm I thought we said we were going to use ible since that's what Google uses in their API ?